docs/blog/2018-1-25-building-a-site-with-react-and-contentful/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Building a Site with React and Contentful
+date: "2018-01-25"
+author: "Shannon Soper"
+---
+
+A poster of Starry Night by Vincent Van Gogh followed me to each new dorm room in college because I bought it the first year, and kept it until it ripped beyond repair. My sister (a professional artist) and I continuously debate the merits and demerits of reprinting great works of art like Van Gogh’s masterpiece. As much as I revere the original, I also love the fact that printing technology makes it easy to put Starry Night on tote bags, coffee mugs, posters, t-shirts, and more because it became more accessible to a wide range of people (without advanced printers and computers, the original painting could only be replicated and shared with more people through painstaking replication via the original medium: oil painting on a canvas).
+
+![Starry Night painting by Vincent Van Gogh](starry-night.jpeg))
```

Comment:
You have extra `)` at the end of line which will end up as text in new line.

Reply:
Thank you @pieh!!! Very helpful and thanks for catching that.